LICENSE.txt

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2012-2023 Scott Chacon and others
```

Comment:
That's neither my nor your name

Reply:
Also add `MIT License` at the top please.

Reply:
aye! Fixed